### PR TITLE
login-status-servlet

### PR DESCRIPTION
**Pull request by josh8551021**
Added user login text to website.
First commit: ace7ade6.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/UserData.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.google.sps.servlets;
+
+class UserData {
+    private boolean isLoggedIn = false;
```
Comment (tzsebe) [inline portfolio/src/main/java/com/google/sps/servlets/UserData.java]: I'd make all of these fields `private final ...`

File: portfolio/src/main/java/com/google/sps/servlets/UserData.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    private String email;
+
+    UserData(String email) {
+        if (!email.isEmpty()) {
```
Comment (tzsebe) [inline portfolio/src/main/java/com/google/sps/servlets/UserData.java]: While this works with your current logic, I think it's actually too constraining. It'd be best if you'd just pass in `isLoggedIn` as a separate variable. After all, a user that's NOT logged in could still have user data (consider the case of an admin looking up some users on some admin UI).

**Review by tzsebe — APPROVED** [review]
Nice work! No need to re-review changes before you submit this, just change whenever.

File: portfolio/src/main/java/com/google/sps/servlets/UserData.java (hunk 1 above)
Comment (josh8551021) [reply portfolio/src/main/java/com/google/sps/servlets/UserData.java]: Done.

File: portfolio/src/main/java/com/google/sps/servlets/UserData.java (hunk 2 above)
Comment (josh8551021) [reply portfolio/src/main/java/com/google/sps/servlets/UserData.java]: Done.

**Review by mkmg — APPROVED** [review]
Nice work!

**Pull request merged**